README.md

```diff
@@ -56,7 +71,7 @@
 	  ....................
 	  127.0.0.1    domain1000
 
-* This would point the sites `http://domain1:8880/index.html` to the original site generated at `http://localhost:8880/index.html`.
+* This would point the sites `http://domain1:8880/index.html` to the original site generated at `http://<destination_ip>:8880/index.html`.
```

Comment:
Looks like this is the only place where ``destination_ip`` is mentioned, can ``localhost`` be used?
```suggestion
* This would point the sites `http://domain1:8880/index.html` to the original site generated at `http://localhost:8880/index.html`.
```